<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Line_Loader_Widget</class>
 <widget class="QWidget" name="Line_Loader_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="dataNameLayout">
     <item>
      <widget class="QLineEdit" name="data_name_text">
       <property name="text">
        <string>line_1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_data_name">
       <property name="text">
        <string>Line Name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="loaderTypeLayout">
     <item>
      <widget class="QLabel" name="label_loader_type">
       <property name="text">
        <string>Loader Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="loader_type_combo">
       <item>
        <property name="text">
         <string>HDF5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CSV</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LMDB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Binary</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="loaderTypeSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stacked_loader_options">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="HDF5LineLoader_Widget" name="hdf5_line_loader"/>
     <widget class="CSVLineLoader_Widget" name="csv_line_loader"/>
     <widget class="LMDBLineLoader_Widget" name="lmdb_line_loader"/>
     <widget class="BinaryLineLoader_Widget" name="binary_line_loader"/>
    </widget>
   </item>
   <item>
    <widget class="Scaling_Widget" name="scaling_widget" native="true"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HDF5LineLoader_Widget</class>
   <extends>QWidget</extends>
   <header>Lines/HDF5/HDF5LineLoader_Widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CSVLineLoader_Widget</class>
   <extends>QWidget</extends>
   <header>Lines/CSV/CSVLineLoader_Widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LMDBLineLoader_Widget</class>
   <extends>QWidget</extends>
   <header>Lines/LMDB/LMDBLineLoader_Widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BinaryLineLoader_Widget</class>
   <extends>QWidget</extends>
   <header>Lines/Binary/BinaryLineLoader_Widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Scaling_Widget</class>
   <extends>QWidget</extends>
   <header>Scaling_Widget/Scaling_Widget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
